src/maptm/src/maptm_config.c

```diff
@@ -524,14 +471,6 @@ bool config_mapt(void)
     snprintf(ipv6prefix, sizeof(ipv6prefix), "%s/%d", MaptConf->ipv6prefix, MaptConf->prefix6len);
     snprintf(ipv4PublicAddress, sizeof(ipv4PublicAddress), "%s/%d", MaptConf->ipv4PublicAddress, MaptConf->prefix4len);
 
-    // Enable firewall
-    if (!maptm_ovsdb_nfm_rules(true))
```

Comment:
After the proposed change, a `maptm_ovsdb_nfm_rules(true)` call isn't present anywhere.

Has this been tested and it works?